src/config-file.ts

```diff
@@ -66,3 +68,14 @@ export function checkIfConfigExists() {
 
   return fs.existsSync(configPath);
 }
+
+export function getConversationStoragePath() {
```

Comment:
Move this function to a separate file like, `save-chat.ts` or similar, as it has little to do with config.

Reply:
or to `file-utils.ts`

Reply:
Done